docs/resources/gmail_send_as_alias.md

```diff
@@ -0,0 +1,80 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "googleworkspace_gmail_send_as_alias Resource - terraform-provider-googleworkspace"
+subcategory: ""
+description: |-
+  Gmail Send As Alias resource in the Terraform Googleworkspace provider. Please ensure the Gmail API is enabled for your workspace and that the user being configured has a Gmail license.
+---
+
+# googleworkspace_gmail_send_as_alias (Resource)
+
+Gmail Send As Alias resource in the Terraform Googleworkspace provider. Please ensure the Gmail API is enabled for your workspace and that the user being configured has a Gmail license.
+
+## Example Usage
+
+```terraform
+data "googleworkspace_user" "example" {
+  primary_email = "user.with.gmail.license@example.com"
+}
+
+resource "googleworkspace_user" "alias" {
+  primary_email = "alias@example.com"
+  password      = "34819d7beeabb9260a5c854bc85b3e44"
+  hash_function = "MD5"
+
+  name {
+    family_name = "Scott"
+    given_name  = "Michael"
+  }
+}
+
+resource "googleworkspace_gmail_send_as_alias" "test" {
+  primary_email = data.googleworkspace_user.example.primary_email
+  send_as_email = googleworkspace_user.alias.primary_email
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- **primary_email** (String) User's primary email address.
+- **send_as_email** (String) The email address that appears in the 'From:' header for mail sent using this alias.
+
+### Optional
+
+- **display_name** (String) A name that appears in the 'From:' header for mail sent using this alias. For custom 'from' addresses, when this is empty, Gmail will populate the 'From:' header with the name that is used for the primary address associated with the account. If the admin has disabled the ability for users to update their name format, requests to update this field for the primary login will silently fail.
+- **id** (String) The ID of this resource.
```

Comment:
Should this be read-only?

Reply:
Oh I didn't add ID in the schema because it's a composite ID and not something people should really be interpolating? I guess it gets added no matter what?